qa/setup_packages.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+from __future__ import print_function, division
+import argparse
+import sys
+try:
+    import pip._internal.pep425tags as p
+except:
+    import pip.pep425tags as p
+try:
+    # For Python 3.0 and later
+    from urllib.request import urlopen, HTTPError, Request
+except ImportError:
+    # Fall back to Python 2's urllib2
+    from urllib2 import urlopen, HTTPError, Request
+
+# keeps names of all required packages as a dict key
+# required versions are list, to use default just put None
+# instead of version number, direct link can be used
+# put {0} in pacage link as a placeholder for python pip package version (i.e. cp27-cp27mu-linux_x86_64)
+packages = {"numpy" : ["1.11.1"],
+            "opencv-python" : ["3.1.0"],
+            "mxnet-cu90" : ["1.3.0b20180612"],
+            "tensorflow-gpu" : ["1.7", "1.8", "1.9", "1.10.0rc0"],
+            "pytorch" : ["http://download.pytorch.org/whl/cu90/torch-0.4.0-{0}.whl"]
```

Comment:
There was mistake - it should be torch, not pytorch. I just fixed it recently with PyTorch example rework.

Reply:
Will fix, thanks